src/ontology/cl-edit.owl

```diff
@@ -30811,12 +30813,12 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16230044") rdfs:comment
 AnnotationAssertion(rdfs:label obo:CL_4030025 "renal cortical fibroblast")
 EquivalentClasses(obo:CL_4030025 ObjectIntersectionOf(obo:CL_0000057 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005270)))
 
-# Class: obo:CL_4030026 (BEST4+ intestinal epithelial cell, human)
+# Class: obo:CL_4030026 (BEST4+ enterocyte)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0002-7431-4139>) Annotation(oboInOwl:hasDbXref "PMID:24223998") Annotation(oboInOwl:hasDbXref "PMID:35176508") obo:IAO_0000115 obo:CL_4030026 "An absorptive epithelial cell of the human intestine expressing bestrophin-4 (BEST4) calcium-activated ion channels.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0002-7431-4139>) Annotation(oboInOwl:hasDbXref "PMID:24223998") Annotation(oboInOwl:hasDbXref "PMID:35176508") obo:IAO_0000115 obo:CL_4030026 "An enterocyte of the human intestine expressing bestrophin-4 (BEST4) calcium-activated ion channels.")
 AnnotationAssertion(terms:date obo:CL_4030026 "2022-09-07T12:34:20Z"^^xsd:dateTime)
-AnnotationAssertion(rdfs:label obo:CL_4030026 "BEST4+ intestinal epithelial cell, human")
-SubClassOf(obo:CL_4030026 obo:CL_0000677)
+AnnotationAssertion(rdfs:label obo:CL_4030026 "BEST4+ enterocyte")
+SubClassOf(obo:CL_4030026 obo:CL_0000584)
```

Comment:
consider to add the following logical definition to the cells expressing BEST4+
`and expresses some bestrophin-4`